src/modules/auth/components/login/loginForm/loginForm.tsx

```diff
@@ -28,7 +32,12 @@ export function LoginForm() {
   }
 
   function onSubmit(data: LoginFormData) {
-    console.log(data);
+    axiosInstance
+      .post('/auth/login', data)
+      .then(() => navigate('/'))
+      .catch((err) => {
+        setError('email', { message: err.response.data.message });
```

Comment:
What if an error is not only about email?

Reply:
Any error coming from the server will be shown to the user. If you take a look at `getFirstErrorMessage` function, it displays the first non-empty error that is present across all the input fields. I could create a separate field for server response errors, but since we have a unified error message box, I decided it was unnecessary.